telegram/utils/warnings.py

```diff
@@ -29,9 +31,22 @@ class TelegramUserWarning(UserWarning):
     __slots__ = ()
 
 
+class PTBRuntimeWarning(PTBUserWarning, RuntimeWarning):
+    """
+    Custom runtime warning class used for warnings in this library.
```

Comment:
```suggestion
    """Custom runtime warning class used for warnings in this library.
```

Reply:
added versionadded directive below it so doesn't matter